### ISSUE TITLE
Support to macOS

### ISSUE THREAD

**Issue by adhenrique** [x42/scarlett-mixer#33]
I'm looking for some software like scarlett-mixer, but with macOS support.

Actually my need is simple: I have an 8i6 but I want to find a way to access the internal API to modify the gain of line inputs in rear panel.

Can anybody help me?

**Comment by tabowling** [comment]
Why not use the Focusrite Control app that comes with the product?
https://downloads.focusrite.com/focusrite/scarlett-3rd-gen/scarlett-8i6-3rd-gen

I don't want to speak for the developers of this project, but it seems to be tightly coupled to the Linux kernel and solving the need that Focusrite is not providing for the Linux platform.

**Comment by adhenrique** [comment]
@tabowling  yes I know it's coupled to Linux. Helped me do some tests on the platform.

However, Focusrite Control does not provide any way to adjust the gain of rear inputs. I figured that asking here, someone could help me in some way.

**Comment by adhenrique** [comment]
@x42 thanks for the links...

Now, I'm getting a permission error: Access Denied (insufficient permissions).

My env
Os: macOs Monterey
Python: v3.10.6

Am I doing something wrong?
